WordPress/pattern-directory
# Theme: Space out context bar on search page.

This PR updates the pattern search in order to add padding to push the context category bar down.


### Screenshots

| Before | After |
| --- | --- |
| ![](https://d.pr/i/vSOMUo.png) | ![](https://d.pr/i/Vj0a53.png) |



### How to test the changes in this Pull Request:

1. Search for a pattern on `/`
2. Expect to see spacing between the breadcrumbs and the category context bar.
